### DISCUSSION TITLE
Generated images in commercial games use

### DISCUSSION THREAD

**Discussion by Nhauer** [carson-katri/dream-textures#68] [Q&A] [answered]
For the images that are generated by the tool, how can the images be used in a commercial sense assuming no external tool is being used? If so, what is the underlying platform (external or not) that is being used to generate these images? Are these images generated covered by the license? 

I would like to use this as a starting point to create textures for use in a game, but I want to avoid any copywrite issues that may arise. I do not see the required info above.

**Comment by carson-katri** [comment, marked as answer]
Check the license for stable diffusion on huggingface for details: https://huggingface.co/spaces/CompVis/stable-diffusion-license
